site/package.json

```diff
@@ -161,14 +159,12 @@
     "msw": "1.3.0",
     "prettier": "3.0.0",
     "protobufjs": "7.2.4",
-    "resize-observer": "1.0.4",
     "rxjs": "7.8.1",
     "ssh2": "1.14.0",
     "storybook": "7.4.0",
     "storybook-addon-react-router-v6": "2.0.0",
     "storybook-react-context": "0.6.0",
     "ts-node": "10.9.1",
-    "ts-proto": "1.157.0",
```

Comment:
Looks like `ts-proto` is used in `make gen` (via `pnpm run gen:provisioner`)  so we might need to add this one back.

Reply:
Yes!